rule/rule.go

```diff
@@ -390,16 +390,18 @@ func (f *File) Format() []byte {
 	return bzl.Format(f.File)
 }
 
-// SortMacro sorts rules in the macro of this File. It doesn't sort the rules if
+// SortMacro sorts rules and loads in the macro of this File. It doesn't sort the rules if
 // this File does not have a macro, e.g., WORKSPACE.
 // This method calls Sync internally.
 func (f *File) SortMacro() {
 	f.Sync()
-	if f.function != nil {
-		sort.Stable(byName{f.Rules, f.function.stmt.Body})
-	} else {
+
+	if f.function == nil {
 		panic(fmt.Sprintf("%s: not loaded as macro file", f.Path))
 	}
+	
+	sort.Stable(loadsByName{f.Loads, f.File.Stmt})
+	sort.Stable(byName{f.Rules, f.function.stmt.Body})
```

Comment:
Can you rename `byName` to `rulesByName` to be consistent with `loadsByName`?

Reply:
Good point, updated. `byIndex` will stay because it can work with any statement type.